model/cachedfeed.py

```diff
@@ -172,6 +169,45 @@ def fetch(cls, _db, lane, type, facets, pagination, annotator,
         # Either there is no cached feed or it's time to update it.
         return feed, False
 
+    @classmethod
+    def calculate_max_age(cls, _db, lane, type):
+        """Helper method to calculate the proper cache time for
+        a WorkList.
+        """
+        from ..opds import AcquisitionFeed
+        from ..lane import Lane
+        # The cache time might come from a site-wide database setting
+        # or from the class definition.
+        database_default = None
+        if type == cls.GROUPS_TYPE:
+            database_default = AcquisitionFeed.grouped_max_age(_db)
```

Comment:
What's the point of keeping a setting for this if we're never going to use it?

Reply:
I guess this is used for WorkLists that aren't lanes, but one of those is the top-level WorkList for a library, which should also be cached forever.

Reply:
And looks like that has a class setting of 2 weeks.

Reply:
Since WorkList has a class setting, this database default will never be used unless a subclass of WorkList resets it to None.

Reply:
I think we probably should get rid of the grouped_max_age setting.

I also think we should either get rid of ungrouped_max_age, or set Lane.MAX_CACHE_AGE to None so that the setting will take effect more often.

Should we do this now?

Reply:
I'd say so, I kind of doubt anyone uses these settings unless it's for development or testing.

Reply:
OK, I took the whole thing out.

There's more we could do -- theoretically you can always derive `type` from the WorkList + the Facets, but I don't want to add that complexity in this branch.

Reply:
I think they're still in config.py. Otherwise, looks good!